src/domain/community/inviteContributors/InviteVCsDialog.tsx

```diff
@@ -52,7 +60,7 @@ const InviteVCsDialog = ({ open, onClose }: InviteContributorDialogProps) => {
   const [bokProfile, setBoKProfile] = useState<BasicSpaceProps>();
 
   const fetchVCs = async () => {
-    let acc = await getAvailableVirtualContributors('', false);
+    let acc = await getAvailableVirtualContributors('');
```

Comment:
Let's convert these two variables to `const` since we're not reassigning new values to them.
Also, I see in my editor we don't have a lint rule for using `let` when not reassigning a new value to it. I suggest we add one.